pkg/descriptions.json

```diff
@@ -1654,6 +1654,13 @@
         ]
     },
     {
+        "Name": "System.Runtime.CompilerServices.Unsafe",
```

Comment:
Ahh good point. They are (mostly) sorted, and this is no longer in the right place.